cmd/cmd.go

```
@@ -17,7 +20,9 @@ func main() {
 func run() error {
 	ctx := context.Background()
 
-	if err := server.Run(ctx, "./config.yaml"); err != nil {
+	configor.Load(&config.Conf, "config.yaml")
```

golangci-lint (1.62.2)

Error return value of `configor.Load` is not checked

(errcheck)